icicle-compiler/test/cli/repl/t30-sea/expected

```diff
@@ -1082,8 +1057,7 @@ if (eq#
         if (doubleIsValid# conv-171)
         {
             flat-118-simpflat-291 =w ExceptNotAnError
-            flat-118-simpflat-292 =w sqrt#
-                                       flat-110-simpflat-288
+            flat-118-simpflat-292 =w conv-171
```

Comment:
this is an important one - duplicating `sqrt`

Reply:
:savage: